cli/tests/pcluster/aws/test_cfn.py

```diff
@@ -20,6 +23,12 @@ def boto3_stubber_path():
     return "pcluster.aws.common.boto3"
 
 
+@pytest.fixture(autouse=True)
+def reset_aws_api():
+    """Reset AWSApi singleton to remove dependencies between tests."""
+    AWSApi._instance = None
```

Comment:
Let's move it to conftest.py, to be used by all the tests.

Reply:
I was blocked by circular imports. As discussed, we decided to keep it here instead of spending more time investigating